ibis/backends/polars/compiler.py

```diff
@@ -1203,9 +1203,19 @@ def execute_union(op, **kw):
     return result
 
 
+def bit_cast_uint64_to_int64(uint: pl.Series) -> pl.Series:
```

Comment:
It feels like there should be a much simpler way to do this or just sidestep the problem completely...